provider-directory-tests/src/main/docker/entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+set -euo pipefail
+# =~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=
+
+init() {
+  test -n "${K8S_ENVIRONMENT}"
+
+  if [ -z "${SENTINEL_BASE_DIR:-}" ]; then SENTINEL_BASE_DIR=/sentinel; fi
+  cd $SENTINEL_BASE_DIR
+
+  SYSTEM_PROPERTIES=()
+
+  if [ -z "${SENTINEL_ENV:-}" ]; then SENTINEL_ENV=$K8S_ENVIRONMENT; fi
+  if [ -z "${CHAPI_URL:-}" ]; then CHAPI_URL=https://$K8S_LOAD_BALANCER; fi
+  if [ -z "${DQ_AVAILABLE:-}" ]; then DQ_AVAILABLE="true"; fi
+  if [ -z "${VFQ_AVAILABLE:-}" ]; then VFQ_AVAILABLE="true"; fi
```

Comment:
remove `DQ_AVAILABLE` and `VFQ_AVAILABLE` and replace `CHAPI_URL` with `PDAPI_URL`